tests/manual/addonStore.md

```diff
@@ -30,6 +30,12 @@ Add-ons can be filtered by display name, publisher and description.
 1. Enable the "Include incompatible add-ons" filter
 1. Ensure add-ons with status "incompatible" are included in the list with the available add-ons.
 
+### Sorting the add-ons list by column
+1. Open the Add-on Store
+1. Jump to the sort by column field (`alt+m`)
```

Comment:
can you add the alternative step (for sighted users/mouse nav) to click the column headers

Reply:
Done